packages/espressocash_app/lib/features/payment_request/widgets/request_success.dart

```diff
@@ -48,14 +40,19 @@ class RequestSuccess extends StatelessWidget {
       content: CpContentPadding(
         child: Column(
           children: [
-            CpTimeline(
-              status: CpTimelineStatus.success,
-              items: [
-                requestCreated,
-                moneyReceived,
-              ],
-              active: 1,
-              animated: false,
+            FutureBuilder(
+              future: request.payRequest.cryptoAmount(
```

Comment:
Never create future in `build` method – in the worst case it can be create with every frame.